src/css/editor.scss

```diff
@@ -1,50 +1,320 @@
-@import 'common';
+@import 'common/common';
```

Comment:
We'll need to split this file component by component using preferably something like CSS Modules.

now it's IMO too big and its readability (as in - specificity) is low

Reply:
(to be done during refactor)